src/components/ClearError.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useCallback } from 'react';
+
+import { TodoErrors } from '../types/TodoErrors';
+import { Action } from './TodoContext';
+
+export const useAutoClearError = (
```

Comment:
👍 
But it's a hook, not component, so the filename shouldn't be in PascalCase

I'd recommend name with `use` at the beginning to highlight there is hook